mod-data-import-converter-storage-server/src/main/java/org/folio/dao/snapshot/ProfileSnapshotDaoImpl.java

```diff
@@ -69,6 +69,7 @@ public ProfileSnapshotDaoImpl(@Autowired PostgresClientFactory pgClientFactory)
           snapshotItem.setDetailId(jsonItem.getString("detail_id"));
           snapshotItem.setDetailType(ContentType.fromValue(jsonItem.getString("detail_type")));
           snapshotItem.setDetail(jsonItem.getJsonArray("detail").getList().get(0));
+          snapshotItem.setOrder(jsonItem.getInteger("detail_order"));
```

Comment:
do we have place where this field is set? I don't see it in this PR

Reply:
I think that this field already received from sql-script